### PR TITLE
Added myself to people.yml and made folder for self (ASR)

### PR DESCRIPTION
**Pull request by anu1217**
Added self to people.yml document

### CODE REVIEW

File: _data/people.yml [hunk 1]
```diff
@@ -58,7 +58,10 @@
     url: "dok"
     role: "PhD Student"
     image: Dean.jpg
-
+  - name: "Anupama Rajendra"
+    url: "asr"
+    role: "PhD Student"
+    image: asr.jpg
```
Comment (gonuke) [inline _data/people.yml:64]: Probably want a blank line after this one.

**Review by gonuke — CHANGES_REQUESTED** [review]
Thanks for starting this update @anu1217 - please also make sure to include pages that describe you and the photo.

File: community/people/asr/index.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+layout: person
+title: Anupama Rajendra
+image: asr.jpg
+role: PhD Student
+email: 
+  - asrajendra@wisc.edu
+room: "3rd Floor, Engineering Research Building"
+services:
+    github: anu1217
+    linkedin: anupama-rajendra-751371190
```
Comment (MicahGale) [inline community/people/asr/index.md:11]: Do you have an ORCID?

**Review by MicahGale — COMMENTED** [review]
I think this just needs to be tested prior to deploying. 

@gonuke Do you not have a CI test build for the pedagogical value of learning docker?

File: community/people/asr/index.md (hunk 2 above)
Comment (anu1217) [reply community/people/asr/index.md:11]: Added ORCID ID

**Comment by anu1217** [comment]
There is a test with docker in the instructions document - tried it and everything looks as intended.

**Comment by gonuke** [comment]
> I think this just needs to be tested prior to deploying. 
> 
> 
> 
> @gonuke Do you not have a CI test build for the pedagogical value of learning docker?

Turns out to be non-trivial to do CI testing of Jekyl sites! I'd love to do testing, which would generally require posting some rendering of the site to some secret web location and then ensuring it is removed when done, etc... 

I'll have to see if anyone's come up with a reliable GitHub action for all that.

**Review by gonuke — APPROVED** [review]
Looks great @anu1217 !  Thanks for adding this.

**Pull request merged**